### PR TITLE
refresh AuthoringTests.md

### PR DESCRIPTION
**Pull request by kivikakk**
Fix Markdown formatting and some old references.

[Rendered](https://github.com/kivikakk/VFSForGit/blob/test-doc-refresh/AuthoringTests.md).

### CODE REVIEW

File: AuthoringTests.md [hunk 1]
```diff
@@ -4,55 +4,56 @@
...
+
+    For any test fixture (a fixture is the same as a class in NUnit) that derives from this class, we create an enlistment and mount VFS for Git before running
 	any of the tests in the fixture, and then we unmount and delete the enlistment after all tests are done (but before any other fixture runs).  If you need
 	to write a sequence of tests that manipulate the same repo, this is the right base class.
 
-3. TestsWithEnlistmentPerTestCase
-	Derive from this class if you need a brand new enlistment per test case.  This is the most reliable, but also most expensive option.
+3. `TestsWithEnlistmentPerTestCase`
+
+   Derive from this class if you need a brand new enlistment per test case.  This is the most reliable, but also most expensive option.
```
Comment (kewillford) [inline AuthoringTests.md]: One other thing that would should add here is the use of `GitRepoTests` and use it if you want to test a git command and have the results of a GVFS repo compared against a regular git clone repo.

Comment (derrickstolee) [reply AuthoringTests.md]: Unfortunately, the repo containing `GitRepoTests` is not public, right? (It contains specific branch and commit names for private, large repos.) So, that is something for our team wiki and not the public docs.

Comment (kewillford) [reply AuthoringTests.md]: The `GitRepoTests` is an `abstract` class in the functional tests that is used to run git commands and compare the output of the git command on GVFS and on a regular git clone.  [This is all public](https://github.com/Microsoft/VFSForGit/blob/655ba9de69843c3b64e20cecaaca2c43bb4466fa/GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs#L14) All the tests that are in the `Tests/GitCommands` folder use this base class.

Comment (derrickstolee) [reply AuthoringTests.md]: Neat! Consider me corrected. I was not familiar with that.

Comment (kewillford) [reply AuthoringTests.md]: We should probably rename it to something like `TestWithGitControlRepo` to match the other base class names and avoid the confusion.

**Pull request merged**